app/services/hyrax/collections/migration_service.rb

```diff
@@ -30,7 +30,7 @@ def self.migrate_all_collections
       # @param collection [::Collection] collection object to be migrated
       def self.migrate_collection(collection)
         return if collection.collection_type_gid.present? # already migrated
-        collection.collection_type_gid = Hyrax::CollectionType.find_or_create_default_collection_type.gid
+        collection.collection_type_gid = Hyrax::CollectionType.find_or_create_default_collection_type.to_global_id.to_s
```

Comment:
It looks like the `.to_s` here is unnecessary because it is being called in `#collection_type_gid=`

Reply:
ty!